### DISCUSSION TITLE
Tell me about your work! - あなたの作品を教えてください！

### DISCUSSION THREAD

**Discussion by pafuhana1213** [pafuhana1213/KawaiiPhysics#65] [Show and tell - 作品紹介]
いつもKawaii Physicsプラグインを使ってくれてありがとうございます！
Thank you for always using the Kawaii Physics plugin!

おかげさまで年々使ってくれている人が増えている気がします。
しかし、私はTwitterと自腹で購入したゲームのライセンスからしか判断できないので、
せっかく使ってくれてるのに見落としている作品もあると思います。
Thanks to you, I feel that the number of people using it is increasing yearly. However, I can only judge from Twitter and the licenses of the games I bought with my own money, so I think there are works I have overlooked even though they are using it.

そのため、もし良ければ KawaiiPhysicsを使った作品を教えてほしいです。
あなたの作品でもいいですし、他の方のものでもOK。シャイな人はフォームからどうぞ！
Therefore, if possible, I would like you to tell me about works and projects using KawaiiPhysics. It can be your work or someone else's work. If you are shy, please use the form! If you are nervous, please use the following form!
https://forms.gle/fkkbsZYPhdgdfWQXA

ちなみに、私が既に把握している作品はこちらの実績ページにまとめています。
By the way, here's a summary of what I already know.
https://github.com/pafuhana1213/KawaiiPhysics/wiki/%E6%8E%A1%E7%94%A8%E5%AE%9F%E7%B8%BE-Adoption

Thanks!

**Comment by fenglengshun** [comment]
This is also in Granblue Fantasy Versus Rising! The game looks great!

![image](https://github.com/pafuhana1213/KawaiiPhysics/assets/26621899/c4e5bc22-47c2-41de-89c0-d40cc86eba5d)

**Reply by pafuhana1213** [reply]
Wow!!! It's AMAZING NEWS!!!
Thanks  a lot for your sharing!!!
https://twitter.com/pafuhana1213/status/1723686430378258465

**Reply by Zharay** [reply]
Just beat Stellar Blade and saw this in the credits! Congrats!
![image](https://github.com/pafuhana1213/KawaiiPhysics/assets/4379473/6e70cd27-af80-4bb7-b71b-ee606cadf029)
Time to see where I can use your work in my own projects.

**Reply by pafuhana1213** [reply]
WTF!? It's great news!!!!
Thanks a lots!
https://x.com/pafuhana1213/status/1786318851162181974?s=46&t=gSGqxyg5upjiuoFCPZjWew

**Reply by Yibby** [reply]
Saw it in Stellar Blade and had to google it because of the name 💯
